README.md

```diff
@@ -19,6 +19,7 @@ resources:
     access_key_id: {{aws-access-key}}
     secret_access_key: {{aws-secret-key}}
     bucket: {{aws-bucket}}
+    filter: <optional, see note below>
```

Comment:
What if we made it more general-purpose (as `options`), so users could pass in anything from http://docs.aws.amazon.com/cli/latest/reference/s3/sync.html#options?